raiden/messages.py

```diff
@@ -1831,6 +1831,21 @@ def verify_request_monitoring(
             and recover(reward_proof_data, self.reward_proof_signature) == requesting_address
         )
 
+class UnlockRequest(Message):
```

Comment:
Lets rename this to `UnlockLightRequest`, just to know that this class is used on light client scenarios.

Reply:
this is outdated